### DISCUSSION TITLE
model selection switch

### DISCUSSION THREAD

**Discussion by adikx5** [EdgeTX/edgetx#3431] [Ideas]
Would it be possible to select the model on the switch and switch between them "imendently"? 
I use the TX16S radio for trucka, construction vehicles and tanks models. Currently, to switch between modals, I have to press MDL, select a model and confirm "select" - with FrSky system and failsafe set to "no pulse" it works very well. I would like (as well as my friends) to be able to switch between models faster - by assigning a given model to a switch. 
Is there any chance to implement such an option? (or maybe LUA script?)
I know FlySky in radium Paladin has this option - it would be a very nice option for "non-aviators".
Not only aviators use EdgeTX :)
It would be great if this could be assigned to this six-way switch.
![339064543_236397235585520_60810560207003893_n](https://user-images.githubusercontent.com/104722932/229599754-859b14f8-f3c7-4238-9985-5829a55540aa.jpg)

**Comment by scooper1** [comment]
I think this would be a great idea for quick switch between models implemented with a confirm dialog on screen maybe a favorite model number assignment related to switch
